config.json

```diff
@@ -180,6 +180,13 @@
         "text formatting",
         "preprocessor (x-macros in test)"
      ]
+  }, {
+     "difficulty": 1,
```

Comment:
I think it is the same as clock -- I will make them both a 2
 -- ratings are tough - you have to leave room for the Russian or American skater that is up last...  someone has to get a 10